### PR TITLE
Add HMAC

### PR DESCRIPTION
**Pull request by ct-clearhaus**
RFR @soltveit @jsmpereira

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -81,15 +81,49 @@ Number  Text
 
 Requests can optionally be signed.
 
-The signature is an RSASSA-PKCS1-v1_5 signature of the body represented in Hex.
-The signee must be identified by the signing API-key. Both should be provided
-in a `Signature` header together with `RS256-hex` identifying the signing
-algorithm:
+The signature is an RSA or HMAC signature represented in Hex of the HTTP body.
```
Comment (jsmpereira) [inline source/index.md]: Reads a bit weird to me.
How about like before: "The signature is an RSA or HMAC signature of the HTTP body represented in Hex."

**Comment by jsmpereira** [comment]
Is it relevant to mention somewhere that singing a request allows you to **_not_** send the CSC?

File: source/index.md (hunk 1 above)
Comment (ct-clearhaus) [inline source/index.md]: I agree. But it's possible to read like "a signature of something; that something is the HTTP body represented in Hex". How one then would represent the HTTP body in Hex ... I dunno.

**Comment by ct-clearhaus** [comment]
> Is it relevant to mention somewhere that singing a request allows you to not send the CSC?

Nah. Because it may also allow you to let others use your card token. It's not a "all or nothing" neither wrt. CSC nor wrt. cross card usage, so I think we better leave it minimal and let those that care know.

File: source/index.md (hunk 1 above)
Comment (jsmpereira) [inline source/index.md]: I agree as well :stuck_out_tongue:

**Comment by ct-clearhaus** [comment]
That's better IMHO. Agree @jsmpereira?

**Comment by jsmpereira** [comment]
:+1:

File: source/index.md [hunk 2]
```diff
@@ -81,15 +81,50 @@ Number  Text
...
+
+```
+OpenSSL::HMAC.hexdigest('sha256', hmac_secret, body)
+```
+
+
+### RSA signature
+
+The RSA signature is an RSASSA-PKCS1-v1_5 signature of the body. It is
+represented in Hex.
```
Comment (jsmpereira) [inline source/index.md]: We had the answer right here :stuck_out_tongue:

**Comment by ct-clearhaus** [comment]
@soltveit Will you deploy soon?

**Pull request merged**